### PR TITLE
to make update

### PR DESCRIPTION
**Pull request by obaidofj**
first upload , then i will change the code to add update and sort

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        showNotes();
+        return;
+      } else {
+        showNotes();
+
+        let hid = document.querySelectorAll("#div1" + i + ",#div2" + i);
+
+        for (let i = 0; i < hid.length; i++) {
+          hid[i].style.display = "none";
+        }
```
Comment (saharAdem) [inline script.js:75]: you can create a parent div for these two divs and then hide the parent div instead of using for loop.

**Comment by saharAdem** [comment]
GREAT JOB, However, try to write more convenient names for your variables

**Review by saharAdem — APPROVED** [review]

File: script.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+let notes = storedNotes ? storedNotes : [];
+let list = document.getElementById("list");
+let upd = 0,
+  targ = "";
+
+document.getElementById("add-btn").addEventListener("click", function () {
+  let title = document.getElementById("title").value;
+  let description = document.getElementById("description").value;
+  //console.log(description, "des");
```
Comment (Rahaf-96) [inline script.js]: remove unnecessary comments, please

File: script.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+  if (title === "") {
+    alert("pleas enter the title of the note");
+  } else {
+    notes.push({ title: title, description: description });
+    document.getElementById("title").value = "";
+    document.getElementById("description").value = "";
+
+    console.log(notes, "notes");
```
Comment (Rahaf-96) [inline script.js]: remove `console.log`

File: script.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
+let storedNotes = JSON.parse(localStorage.getItem("notes"));
+
+let notes = storedNotes ? storedNotes : [];
+let list = document.getElementById("list");
+let upd = 0,
+  targ = "";
+
+document.getElementById("add-btn").addEventListener("click", function () {
```
Comment (Rahaf-96) [inline script.js:8]: if the user press `space` as an input, it will add an empty list item. you should handle this case too.

File: script.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        inpTitle.setAttribute("id", "inp1");
+        inpTitle.value = note.title;
+
+        let inpDescription = document.createElement("textarea");
+        inpDescription.setAttribute("cols", "60");
+        inpDescription.setAttribute("rows", "8");
+        inpDescription.setAttribute("id", "inp2");
+        inpDescription.value = note.description;
+        let listIt = document.getElementById("li" + i);
+        //console.log(i);
```
Comment (Rahaf-96) [inline script.js]: remove this comment too.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+#add-btn {
+  margin: 20px;
+  padding: 5px 10px;
+  font-size: 15px;
+  border: solid 1px #ccc;
+  width: 150px;
+  margin-top: 30px;
+  /* transform: translate(-50%,-50%); */
```
Comment (Rahaf-96) [inline style.css:46]: remove this comment.

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]

**Review by Rahaf-96 — APPROVED** [review]
Great job!!

**Pull request merged**